### PR TITLE
Business central Advanced settings UI

### PR DESCRIPTION
**Pull request by DhaaraniCIT**
## Summary by CodeRabbit

- **New Features**
  - Introduced new configuration options for automation and customization in the Business Central advanced settings.
  - Enhanced the user interface for the Business Central advanced settings to reflect new features.

- **Improvements**
  - Updated the loading behavior to improve user experience during data retrieval.
  - Improved the display of additional conditions in the Configuration Skip Export component.

- **Style Updates**
  - Added new styles for `.preview-text` and `.preview-box` to enhance visual presentation.

- **Bug Fixes**
  - Fixed an issue where the `isLoading` flag was incorrectly set, affecting the display of loading states.

### CODE REVIEW

**Comment by DhaaraniCIT** [comment]
@coderabbitai review

**Comment by DhaaraniCIT** [comment]
Screenshot 2023-12-11 at 2 45 17 PM
Screenshot 2023-12-11 at 2 45 25 PM

File: src/app/integrations/business-central/business-central-shared/business-central-advanced-settings/business-central-advanced-settings.component.ts [hunk 1]
```diff
@@ -22,7 +23,7 @@ import { AppName, ConfigurationCta } from 'src/app/core/models/enum/enum.model';
 })
 export class BusinessCentralAdvancedSettingsComponent implements OnInit {
 
-  isLoading: boolean = true;
+  isLoading: boolean = false;
```
Comment (ashwin1111) [inline src/app/integrations/business-central/business-central-shared/business-central-advanced-settings/business-central-advanced-settings.component.ts:26]: this should be true by default

**Review by ashwin1111 — APPROVED** [review]

File: src/app/integrations/business-central/business-central-shared/business-central-advanced-settings/business-central-advanced-settings.component.ts (hunk 1 above)
Comment (DhaaraniCIT) [reply src/app/integrations/business-central/business-central-shared/business-central-advanced-settings/business-central-advanced-settings.component.ts:26]: fixed in next pr

**Pull request merged**